src/models/userModel.js

```diff
@@ -419,6 +419,15 @@ const User = new Mongoose.model('User', new Mongoose.Schema ({
         elo: {
             type: Number,
             default: 0
+        },
+        // Fields for first daily win streak goes up to 3 days 
```

Comment:
^ We need to start users with way less than $50,000 for this price range. $300-$500 is probably more reasonable.

Reply:
I think it should be $1,000 at least. The new user's need room for error.

Reply:
Changed it to 1000, my bad lol